devtools/test_dashboard/buttons.js

```diff
@@ -14,31 +14,27 @@ anchor.style.height = '600px';
 anchor.style.width = '1000px';
 
 function plotButtons(plots, figDir) {
-
     Object.keys(plots).forEach(function(plotname) {
-
         var button = document.createElement('button');
 
         button.style.cssFloat = 'left';
         button.style.width = '100px';
         button.style.height = '40px';
-
         button.innerHTML = plotname;
 
-        plotlist.appendChild(button);
+        plotList.appendChild(button);
 
         button.addEventListener('click', function() {
-
             var myImage = new Image();
             myImage.src = figDir + plotname + '.png';
 
             image.innerHTML = '';
             image.appendChild(myImage);
 
+            gd = document.createElement('div');
+            gd.id = 'graph';
```

Comment:
@mdtusz not sure why I didn't do this before, now you can use `Plotly.plot('graph', [], {})` in the test-dashboard.